src/igortregub/lifo/Stack.java

```diff
@@ -0,0 +1,31 @@
+package igortregub.lifo;
+
+import java.util.EmptyStackException;
+
+public class Stack implements StackInterface {
+    private int[] items;
+
+    private int currentItem = -1;
+
+    Stack(int stackSize) {
+        items = new int[stackSize];
+    }
+
+    @Override
+    public void push(int item) throws IndexOutOfBoundsException {
+        if (items.length <= currentItem + 1) {
+            throw new IndexOutOfBoundsException("Stack is full.");
+        }
+
+        items[++currentItem] = item;
```

Comment:
```suggestion
        items[++currentItem] = item;
```
items[currentItem++] = item;